### PR TITLE
Add DB dump download feature to refresh script

### PR DESCRIPTION
**Pull request by contolini**
With this commit, invoking the refresh-data script without any arguments will download the latest DB dump for you (if CFGOV_PROD_DB_LOCATION is defined).

## Testing

```sh
export CFGOV_PROD_DB_LOCATION=http://<our-secret-location-check-ghe>/production_django.sql.gz
./refresh-data.sh
```

It should download `production_django.sql.gz` and then do the normal MySQL import process.

Running `./refresh-data.sh` without any arguments and without CFGOV_PROD_DB_LOCATION defined will show some help text.

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

File: refresh-data.sh [hunk 1]
```diff
@@ -10,15 +10,41 @@
...
+
+# Only attempt to load data if a db dump file is provided
+if [[ -z "$refresh_dump_name" ]]; then
+	echo 'Please download a recent database dump before running this script:
+
+	./refresh-data.sh production_django.sql
+
+Or you can define the location of a dump and this script will download it for you:
+
+	export CFGOV_PROD_DB_LOCATION=http://bucket-name.s3.amazonaws.com/db_dumps/production_django.sql.gz
```
Comment (higs4281) [inline refresh-data.sh]: The path after the generic "bucket-name" is not complete, but maybe that's intentional?

Comment (contolini) [inline refresh-data.sh]: Yeah the real path was so long that it was wrapping to a new line so I truncated it. I'll change this example path to make it clearer that it's fake.

**Comment by higs4281** [comment]
:+1:

**Comment by Scotchester** [comment]
I just tried this out and it worked great!

**Pull request merged**